helpers/tokenHelpers.ts

```diff
@@ -1,13 +1,14 @@
 ﻿import {ServerResponse} from "http";
 import {ParsedUrlQuery} from "querystring";
-import {checkToken} from "../api/sessionClient";
+import {checkToken} from "../pages/api/sessionClient";
 
-export async function assertTokenIsValid(query: ParsedUrlQuery, response: ServerResponse): Promise<void> {
+export async function assertTokenIsValid(query: ParsedUrlQuery, response: ServerResponse): Promise<string> {
     const token = query.token as string;
     const tokenIsValid = await checkToken(token);
-    
+
     if (!tokenIsValid) {
         response.statusCode = 404;
         response.end();
     }
+    return token;
```

Comment:
Doesn't look like we ever use this return value. Is there a need for it?
If not, could you change it back to what we had before?

Reply:
I use it to get candidate statuses